src/app/pcasts/__init__.py

```diff
@@ -24,6 +24,8 @@
 from app.pcasts.controllers.get_listening_history_controller import * # pylint: disable=C0413
 from app.pcasts.controllers.clear_listening_history_controller import * # pylint: disable=C0413
 from app.pcasts.controllers.get_feed_controller import * # pylint: disable=C0413
+from app.pcasts.controllers.update_session_controller import * # pylint: disable=C0413
+from app.pcasts.controllers.sign_out_controller import * # pylint: disable=C0413
```

Comment:
Really nitpicking here, but what are your guys' thoughts on disabling C0413 for good? The error description is: 
Import “%s” should be placed at the top of the module Used when code and imports are mixed

I feel like we don't need a linter to know that imports shouldn't be below code, unless its necessary like in this case (and a ton of other cases, especially with my testing stuff). Not a huge deal, but maybe something to consider

Reply:
Let's talk about this IRL - it has its positives and negatives